docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[xpack-logs-analysis]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+With the {ml} {anomaly-detect} features enabled,
+you can use the *Log rate* page in the Logs app to automatically detect and inspect log anomalies and
+the log partitions in which they occurred.
+This helps you to spot suspicious behavior without significant human intervention --
+no more manually sampling log data, calculating rates, and determining if rates are normal.
+
+*Log rate* automatically highlights periods of time where the log rate is outside expected bounds and therefore may be anomalous.
+You can use this information as a basis for further investigations.
+For example:
+* A significant drop in the log rate might suggest that some some piece of infrastructure has stopped responding,
+and thus we're serving less requests.
+* A spike in the log rate could denote a DDoS attack.
+This may lead to an investigation of IP addresses from incoming requests.
+
+TIP: You can also view log anomalies directly in the <<xpack-ml-anomalies,Machine Learning app>>.
+
+[float]
+[[logs-analysis-create-ml-job]]
+=== Create a machine learning job for logs analysis
+
+To enable log analysis and anomaly detection, you must first create your own {kibana-ref}/xpack-spaces.html[space].
+Within a space, you can create a machine learning job which will carry out the logs analysis.
+
+[role="screenshot"]
+image::logs/images/analysis-tab-create-ml-job.png[Create machine learning job]
+
+1. Select the time range for the analysis.
```

Comment:
Is it maybe worth mentioning this is the "Machine Learning analysis"? With the part about the time range for the analysis below, I'm thinking it might be worth really drilling home the fact these are the parameters that feed directly into ML.